### ISSUE TITLE
As the Open Forest researcher, I want a test plan in preparation for Admin Role usability testing so that we can understand more about user management.

### ISSUE THREAD

**Issue by lisamcmichael** [USDAForestService/fs-open-forest-platform#855]
## Notes
DRAFT!! 
* What are things we should consider when making this story *
- See story 689
- Research should include roles Super Admin (highest permission level  - perhaps Aaron) and Admins with different access levels (e.g., read, write and/or manage)

## Acceptance Criteria
- [x] PO approves usability test plan

## Tasks
User research checklist
- [x] Create research plan (includes test script)
    - [x] Discuss scenarios with Designer/UX architect
- [x] Review plan with PO
- [x] Finalize plan with any changes from PO

## Definition of Done
- [x] Research plan ready for pilot test
- [x] Research plan added to Pinyon/Research folder
- [x] Approved by PO

**Comment by lisamcmichael** [comment]
@ASprinkle, you are assigned to this given your role as "approver".

**Comment by lisamcmichael** [comment]
**9/17, meet with Lisa**
Determine tasks for testing
**9/18, create test plan draft**
Send to Amber/Lisa

**Comment by lisamcmichael** [comment]
The test plan is a stand alone story. It's not blocked by test preparations for recruiting and scheduling activities.

**Comment by ASprinkle** [comment]
@lisamcmichael I "unblocked" this story, and edited the story title to match the goals/objectives vs. including testing etc. :)

**Comment by lisamcmichael** [comment]
I will email this to @ASprinkle and @lisaredux later today, and set up a meeting to review.

**Comment by ASprinkle** [comment]
I have availability all day

**Comment by lisamcmichael** [comment]
Hi @ASprinkle and @lisaredux *Here is Version 1.0 of the research plan. If there isn't already a meeting on our calendars for tomorrow, I'll get it setup. It's on Box/Pinyon.*

https://usfs.box.com/s/iabkf6makntcyb74835n4hdmjmebxwzj

**Comment by lisamcmichael** [comment]
@ASprinkle , I noted that this story is not in the 2019 MVP release.  

Also, @lisaredux  and I met to review the plan.  Lisa and I recalled that "modal windows" would be a large level of effort for Mike and Brian. So for now consider those as placeholders, and, we should consider all of the screens shown in the test plan as placeholders for now.

**Comment by ASprinkle** [comment]
I am approving this test plan as completed with the understanding that some screen shots and nuance may change later.

**Comment by lisamcmichael** [comment]
@ASprinkle and @lisaredux Good morning. 

I am considering creating a new story for the Pilot Testers (Dana and Farrah) so it's separated from 855. I wonder if that is really necessary?

**Comment by lisaredux** [comment]
@lisamcmichael — not sure but it might be easier to enscapsulate the details of the test plan in a Wiki link attached to this story. My two cents.

**Comment by lisamcmichael** [comment]
> @lisamcmichael — not sure but it might be easier to inscapsulate the details of the test plan in a Wiki link attached to this story. My two cents.

I'll follow back up w/ you on this. Thx.

**Comment by ASprinkle** [comment]
@lisamcmichael 855 is "done" so it would be worth creating a story to capture the work for usability testing with Dana Croll and Fara Humphrey.  Any test plans associated with their testing should be housed in the same location as the test plan from this story.  

It also looks like the test plan is housed in Pinyon currently. It should be also posted to our Wiki https://github.com/USDAForestService/fs-open-forest-platform/tree/dev/docs/christmas-trees/usability-testing
